<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_password</class>
 <widget class="QWidget" name="modify_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/qt 图标/修改图标/document-write.ico</activeon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: qlineargradient(spread:pad, x1:0.500273, y1:0.438, x2:0.500727, y2:0.909, stop:0 rgba(190, 154, 124, 255), stop:0.971591 rgba(96, 71, 62, 255));
}

QMessageBox {   border-color:#32435E; border-width:1px; border-radius: 3px;
  min-width:30px; min-height:23px;
  /*font-family:&quot;宋体&quot;;font-size:36px;*/
  border-image:url(./config/qss/style_1/images/messageBox_background.png);
  height: 300px;
  width: 200px;
  border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="text">
      <string>登录用户名：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="text">
      <string>修改密码：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color:rgb(190, 154, 124, 255);</string>
     </property>
     <property name="text">
      <string>当前密码：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">
QPushButton{  
color:rgb(255, 255, 255);
font: 75 10pt &quot;微软雅黑&quot;;
border-style:solid;  
background-color:rgb(255, 85, 0);
border-radius:5px;padding:2px 4px;/*border-radius控制圆角大小*/  
}  
QPushButton:hover{  /*鼠标放上后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background-color:rgb(190, 154, 124, 255);
border:1px;  
border-radius:5px;padding:2px 4px;  
}  
QPushButton:pressed{ /*按下按钮后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                            stop: 0.3 rgb(190,190,190),  
                              stop: 1 rgb(160,160,160));  
border:1px;  
border-radius:5px;padding:2px 4px;  
}  
QPushButton:checked{    /*选中后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                            stop: 0.3 rgb(190,190,190),  
                              stop: 1 rgb(160,160,160));  
border:1px;  
border-radius:5px;padding:2px 4px;  
}  </string>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">
QPushButton{  
color:rgb(255, 255, 255);
font: 75 10pt &quot;微软雅黑&quot;;
border-style:solid;  
background-color:rgb(255, 85, 0);
border-radius:5px;padding:2px 4px;/*border-radius控制圆角大小*/  
}  
QPushButton:hover{  /*鼠标放上后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background-color:rgb(190, 154, 124, 255);
border:1px;  
border-radius:5px;padding:2px 4px;  
}  
QPushButton:pressed{ /*按下按钮后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                            stop: 0.3 rgb(190,190,190),  
                              stop: 1 rgb(160,160,160));  
border:1px;  
border-radius:5px;padding:2px 4px;  
}  
QPushButton:checked{    /*选中后*/  
color:rgb(255, 255, 255);  
min-height:20;  
border-style:solid;  
border-top-left-radius:2px;  
border-top-right-radius:2px;  
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 rgb(226,236,241),   
                            stop: 0.3 rgb(190,190,190),  
                              stop: 1 rgb(160,160,160));  
border:1px;  
border-radius:5px;padding:2px 4px;  
}  </string>
     </property>
     <property name="text">
      <string>放弃修改</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
